README.md

```diff
@@ -45,3 +45,14 @@ with:
   DEB_DISTRO: jammy
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
+
+## Running manually
```

Comment:
this paragraph seems out of scope as you removed the commit that it belong to. Can you remove it?

Reply:
Not sure what you mean here. Remove the whole paragraph? The line preceding it?

Reply:
All current changes in the `README.md` as they describe the state after my commit and are unrelated to this PR.